src/main/java/uk/gov/hmcts/reform/cwrdapi/service/impl/CaseWorkerServiceImpl.java

```diff
@@ -167,13 +166,30 @@ public IdamRolesMappingResponse buildIdamRoleMappings(List<ServiceRoleMapping> s
         }
     }
 
+    /**
+     * Returns the caseworker details based on the id's.
+     * @param caseWorkerIds list
+     * @return CaseWorkerProfile
+     */
+    @Override
+    public ResponseEntity<Object> fetchCaseworkersById(List<String> caseWorkerIds) {
+        long currentTimeMillis = System.currentTimeMillis();
+        List<CaseWorkerProfile> caseWorkerProfileList = caseWorkerProfileRepo.findByCaseWorkerIdIn(caseWorkerIds);
+        log.info("Time taken by fetchCaseworkersById method = " + (System.currentTimeMillis() - currentTimeMillis));
```

Comment:
Is it really required to log the duration ?

Reply:
Removed the logging.